src/main/java/org/seqdoop/hadoop_bam/util/SAMHeaderReader.java

```diff
@@ -70,4 +78,19 @@ public static ValidationStringency getValidationStringency(
 		final String p = conf.get(VALIDATION_STRINGENCY_PROPERTY);
 		return p == null ? null : ValidationStringency.valueOf(p);
 	}
+
+	public static ReferenceSource getReferenceSource(
+			final Configuration conf)
+	{
+		//TODO: There isn't anything particularly CRAM-specific about reference source or validation
+		// stringency other than that a reference source is required for CRAM files. We should move
+		// the reference source and validation stringency property names and utility methods out of
+		// CRAMInputFormat and SAMHeaderReader and combine them together into a single class for extracting
```

Comment:
Agree. Maybe something akin to HTSJDK's ReferenceSource? But for now I think this works fine.